doc/build/sysbuild/index.rst

```diff
@@ -0,0 +1,456 @@
+.. _sysbuild:
+
+Sysbuild (System build)
+#######################
+
+Sysbuild is a higher-level build system that can be used to combine multiple
+other build systems together. For example, you can use sysbuild to build a
+Zephyr application together with the MCUboot bootloader, flash them both onto
+your device, and debug the results.
+
+Sysbuild works by configuring and building a Zephyr application and as many
+additional projects as you want (including none). The additional projects can
+also be Zephyr applications, or they can be some other type of build you want
+to run.
+
+Like Zephyr's :ref:`build system <build_overview>`, sysbuild is written in
+CMake and uses :ref:`Kconfig <kconfig>`. However, sysbuild is **not** the same
+thing as the Zephyr build system. It is a higher level layer that combines one
+or more Zephyr build systems, along with optional additional build systems,
+into a hierarchical build system with shared configuration.
+
+Definitions
+***********
+
+Here are some key concepts used in this document.
+
+Single-image build
+    When sysbuild is used to create and manage just one Zephyr application's
+    build system.
+
+Multi-image build
+   When sysbuild is used to manage multiple build systems. (The word "image" is
+   used because your main goal is usually to generate binary firmware
+   application images from each build system.)
```

Comment:
```suggestion
   When sysbuild is used to manage multiple build systems. 
   The word "image" is used because your main goal is usually to generate the binaries of the firmware
   application images from each build system.
```